Gemfile

```diff
@@ -20,8 +20,6 @@ gem 'jbuilder'
 # For html parsing
 gem 'nokogiri'
 
-gem 'rkerberos'
-
 gem 'net-ssh-krb'
```

Comment:
This is also kerberos related.

This should probably be reflected in Gemfile.lock

Reply:
Can't we just remove it here, and not care about the lock file. Won't the lock file just be regenerated if we maybe remove it?

Reply:
`RUN bundle install` in the Dockerfile should update the Gemfile.lock automatically. If we are to lazy to include the changes now, the updated lockfile will sit in everyone's staging area - basically just postponing the work.

Reply:
Updated the Gemfile.lock, including removing rkerberos